### ISSUE TITLE
snipmate errors out on multiple-choice snippets

### ISSUE THREAD

**Issue by jherazob** [spf13/spf13-vim#35]
First of all, i did the couple of fixes mentioned on the comments of other issues, fixing the order of the VimDebugger functions as mentioned on issue #27, and eliminating the extra directory on the the path of the snipmate-snippets as mentioned on issue #24. 

I create a new html file. I type "skel<tab>", i get this:

```
Se ha detectado un error al procesar function TriggerSnippet..<SNR>32_GetSnippet..<SNR>32_ChooseSnippet:
línea   12
E117: Función desconocida: tlib#input#List
Pulse INTRO o escriba una orden para continuar
Se ha detectado un error al procesar function TriggerSnippet..<SNR>32_GetSnippet..<SNR>32_ChooseSnippet:
línea   12
E15: La expresión no es válida: tlib#input#List('si','select snippet by name',snippet) -1
Pulse INTRO o escriba una orden para continuar
Se ha detectado un error al procesar function TriggerSnippet..<SNR>32_GetSnippet..<SNR>32_ChooseSnippet:
línea   13
E121: Variable sin definir: idx
Pulse INTRO o escriba una orden para continuar
Se ha detectado un error al procesar function TriggerSnippet..<SNR>32_GetSnippet..<SNR>32_ChooseSnippet:
línea   13
E15: La expresión no es válida: idx == -1
Pulse INTRO o escriba una orden para continuar
Se ha detectado un error al procesar function TriggerSnippet..<SNR>32_GetSnippet..<SNR>32_ChooseSnippet:
línea   19
E121: Variable sin definir: idx
Pulse INTRO o escriba una orden para continuar
Se ha detectado un error al procesar function TriggerSnippet..<SNR>32_GetSnippet..<SNR>32_ChooseSnippet:
línea   19
E116: Argumentos inválidos para la función: funcref#Call(a:snippets[keys(a:snippets)[idx]])
Pulse INTRO o escriba una orden para continuar
Se ha detectado un error al procesar function TriggerSnippet..<SNR>32_GetSnippet..<SNR>32_ChooseSnippet:
línea   19
E15: La expresión no es válida: funcref#Call(a:snippets[keys(a:snippets)[idx]])
Pulse INTRO o escriba una orden para continuar
Se ha detectado un error al procesar function TriggerSnippet..<SNR>32_GetSnippet..<SNR>32_ChooseSnippet:
línea   12
E117: Función desconocida: tlib#input#List
Pulse INTRO o escriba una orden para continuar
Se ha detectado un error al procesar function TriggerSnippet..<SNR>32_GetSnippet..<SNR>32_ChooseSnippet:
línea   12
E15: La expresión no es válida: tlib#input#List('si','select snippet by name',snippet) -1
Pulse INTRO o escriba una orden para continuar
Se ha detectado un error al procesar function TriggerSnippet..<SNR>32_GetSnippet..<SNR>32_ChooseSnippet:
línea   13
E121: Variable sin definir: idx
Pulse INTRO o escriba una orden para continuar
Se ha detectado un error al procesar function TriggerSnippet..<SNR>32_GetSnippet..<SNR>32_ChooseSnippet:
línea   13
E15: La expresión no es válida: idx == -1
Pulse INTRO o escriba una orden para continuar
Se ha detectado un error al procesar function TriggerSnippet..<SNR>32_GetSnippet..<SNR>32_ChooseSnippet:
línea   19
E121: Variable sin definir: idx
Pulse INTRO o escriba una orden para continuar
Se ha detectado un error al procesar function TriggerSnippet..<SNR>32_GetSnippet..<SNR>32_ChooseSnippet:
línea   19
E116: Argumentos inválidos para la función: funcref#Call(a:snippets[keys(a:snippets)[idx]])
Pulse INTRO o escriba una orden para continuar
Se ha detectado un error al procesar function TriggerSnippet..<SNR>32_GetSnippet..<SNR>32_ChooseSnippet:
línea   19
E15: La expresión no es válida: funcref#Call(a:snippets[keys(a:snippets)[idx]])
Pulse INTRO o escriba una orden para continuar
```

The expected behavior here should be to offer me a list of the contents of .vim/bundle/snipmate-snippets/snippets/html/skel/ instead of erroring out like that.

Will be looking for a workaround/fix and post it if found.

**Comment by jherazob** [comment]
Fix found: It's missing the prerequisite plugin tlib. To fix it:
     git clone https://github.com/tomtom/tlib_vim.git ~/.vim/bundle/tlib

Now it works. I can't commit a fix where i am right now, but it should be just to add that line on bootstrap.sh

**Comment by sbucek** [comment]
THANK YOU jherazob!!!

**Comment by spf13** [comment]
This has been implemented in the lastest branch (3.0).
